### PR TITLE
Add Homepage and Routes

### PR DESCRIPTION
**Pull request by viru2001**
I have added Homepage and routes

Homepage-compresse

### CODE REVIEW

**Review by bhavyaj12 — APPROVED** [review]
The text and button on top of carousel image looks a little strange because you've made its opacity less I guess, that can be improved for UI.

Great file structure and breakdown of components

File: src/App.js [hunk 1]
```diff
@@ -1,42 +1,13 @@
...
-            rel="noreferrer"
-          >
-            Contribute
-          </a>
-        </div>
-      </header>
-    </div>
+    <>
+      <Navbar />
+      <Router />
```
Comment (ravikumar0403) [inline src/App.js:8]: Read about useRoutes hook

File: src/backend/db/categories.js [hunk 2]
```diff
@@ -1,5 +1,11 @@
 import { v4 as uuid } from "uuid";
-
+import {
+  MonitorCategoryImg,
+  ChairCategoryImg,
+  MouseCategoryImg,
+  HeadphonesCategoryImg,
+  KeyboardCategoryImg,
+} from "../../frontend/assets";
```
Comment (ravikumar0403) [inline src/backend/db/categories.js:8]: use path aliasing here

File: src/frontend/components/Navbar/NavItem.jsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import { Link } from "react-router-dom";
+
+const NavItem = ({ navItemTitle, number, icon, isNumberReqd, navigateTo }) => {
+  return (
+    <Link
```
Comment (ravikumar0403) [inline src/frontend/components/Navbar/NavItem.jsx:5]: Read about NavLink component from react router dom

File: src/frontend/components/Navbar/NavItem.jsx [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+      className="nav-link-icon icon-badge-wrapper m-3 d-flex flex-col align-center text-dec-none"
+      to={navigateTo}
+    >
+      <span className="p-relative">
+        <i className={`fas fa-${icon} text-md`}></i>
+        {isNumberReqd ? (
```
Comment (ravikumar0403) [inline src/frontend/components/Navbar/NavItem.jsx]: You can use && instead of ternary

File: src/frontend/components/Navbar/Navbar.jsx [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+      <section id="nav-menu-without-hamburger" className="d-flex m-2">
+        <nav className="nav-links d-flex align-center">
+          <NavItem
+            isNumberReqd={false}
+            navItemTitle={"Products"}
+            icon={"store"}
+            navigateTo={"/products"}
+          />
+          <NavItem
+            isNumberReqd={true}
```
Comment (ravikumar0403) [inline src/frontend/components/Navbar/Navbar.jsx]: isNumberReqd can be omitted and you can check the condition using number prop

File: src/frontend/pages/index.js [hunk 6]
```diff
@@ -0,0 +1,6 @@
+import { Cart } from "./Cart/Cart";
+import { Home } from "./Home/Home";
+import { Products } from "./Products/Products";
+import { Wishlist } from "./Wishlist/Wishlist";
+
+export { Home, Products ,Cart , Wishlist};
```
Comment (ravikumar0403) [inline src/frontend/pages/index.js]: can export it without using import

**Review by ravikumar0403 — APPROVED** [review]
Code is clean and folder structure is also good. I've added a few comments which can be improved

**Pull request merged**